scripts/apps/authoring/metadata/PlacesService.ts

```diff
@@ -0,0 +1,60 @@
+import {omit} from 'lodash';
+
+export interface IPlacesService {
+    search: (name: string, lang: string, format: 'geonames' | 'dateline') => Promise<Array<{}>>;
```

Comment:
The return types aren't clear in the interface. We would also get much better typing support if it were 2 methods instead:

```
searchGeonames: (name: string, lang: string) => Promise<Array<IGeoName>>;
searchDateline: (name: string, lang: string) => Promise<Array<IDateline>>;
```

Specifying the exact return type is easy and you don't need to do checks to determine the type after you call the function.